examples/vue-app/routes/todos.vue

```diff
@@ -0,0 +1,105 @@
+<script setup>
+import { Head, useData } from "aleph/vue"
+
+const { data, isMutating, mutation } = useData();
+
+async function onChange(todo) {
+  const { id } = todo;
+  const completed = !todo.completed;
+  mutation.patch({ id, completed }, "replace")
+}
+
+async function onSubmit(e) {
+  e.preventDefault();
+  const form = e.currentTarget;
+  const fd = new FormData(form);
+  const message = fd.get("message")?.toString().trim();
+  if (message) {
+    mutation.put({ message }, {
+      // optimistic update without waiting for the server response
```

Comment:
yessss! Supporting of optimistic UI in the context is awesome, nice!

Reply:
TBH,  I mostly copy your code, it look good.